src/io.cpp

```diff
@@ -237,19 +237,7 @@ void IoCore::Shutdown()
 	uv_close(reinterpret_cast<uv_handle_t *>(&this->server), nullptr);
 
 	// Finally, kill off all of the connections with 'fatal' responses.
-	for (const auto conn : this->pool) IoCore::TryShutdown(conn);
-}
-
-/* static */ void IoCore::TryShutdown(const std::shared_ptr<Connection> conn)
-{
-	if (!conn) return;
-
-	auto response = Response(Response::Code::STATE).AddArg("Quitting");
```

Comment:
I presume this is because nothing is ever likely to read this before libuv shuts down their connections?